src/main/resources/schema.graphql

```diff
@@ -23,8 +23,74 @@ input RunsRequest {
   tags: JSON
 }
 
-
 type Mutation {
     startRun(request: RunsRequest!): RunsResponse
     cancelRun(runId: ID!): RunsResponse
 }
+
+type SearchResultInfo {
+    contentCount: String!
+    hasNextFrom: String!
+    totalHits: String!
+}
+
+type EngineParameters {
+    launchDir: String
+    projectDir: String
+    resume: String
+    revision: String
+    workDir: String
+    defaultContainer: String
+    latest: String
+}
+
+type ActiveRun {
```

Comment:
I think I see why you called it ActiveRun ... currently Management already has it's own "Run" entity that is a WES run. Once we separate that into the "Workflow API" I think we can name it just Run here?

Reply:
Yea that was exactly my thinking, I think it makes sense to keep these idea separate (at least for now). The nice thing about this db is we can totally nuke it (provided no runs are active) and do whatever. No need to worry about migrations.

Reply:
Yeah, we have that active run nomenclature in a lot of places though, maybe make a note about the migration plan for it